lib/bloc/explorer/explorer_bloc.dart

```diff
@@ -417,13 +417,30 @@ class ExplorerBloc extends Bloc<ExplorerEvent, ExplorerState> {
           await database.updateVtt(wallet.id, vtt);
         }
       } catch (e) {
-        var vtt = await Locator.instance.get<ApiExplorer>().hash(_vtt.txnHash);
-        if (vtt != null && vtt.jsonMap()['status'] == 'unknown hash') {
-          vtt = _vtt;
-          ValueTransferInfo unknownTransaction = _vtt;
-          unknownTransaction.status = 'unknown hash';
-          await database.updateVtt(wallet.id, unknownTransaction);
+        /// If the getVtt method returns null we enter this catch
+        /// and the vtt has an unknown hash
+
+        /// check the inputs for accounts in the wallet and remove the vtt
+        for (int i = 0; i < _vtt.inputs.length; i++) {
+          Account? account = wallet.accountByAddress(_vtt.inputs[i].address);
+          if (account != null) {
+            account.deleteVtt(_vtt);
+            account.vttHashes.removeWhere((element) => element == _vtt.txnHash);
+            await database.updateAccount(account);
```

Comment:
Could you add this part of the code to a different function?

```suggestion
           account.deleteVtt(_vtt);
            account.vttHashes.removeWhere((element) => element == _vtt.txnHash);
            await database.updateAccount(account);
```